scripts/log-templates/log-config-qa.yaml

```diff
@@ -1,13 +1,11 @@
 # This template is used in scripts to connect to a running cluster.
 
 rotation:
-    logLimit: 2097152 # 2MB
-    keepFiles: 20
+    logLimit: 104857600 # 100MB
+    keepFiles: 2
```

Comment:
If it's for qa only, i would suggest something in between. Say, 50mb and 10 files.

Reply:
QA is confusing name. It's for developers who use `mainnet.sh` and similar stuff.

Reply:
Yep, I know (I'm using it), the point was "we don't care about space since it's for devs/testing only".